configs/components/razor-server.rb

```diff
@@ -73,16 +104,16 @@
   # On upgrade, check to see if these files exist and copy them out of the way to preserve their contents
   pkg.add_preinstall_action ['upgrade'],
     [
-      "[[ -e /etc/razor/config.yaml ]] && mkdir -p /tmp/.razor-server.upgrade && cp /etc/razor/config.yaml /tmp/.razor-server.upgrade/config.yaml",
-      "[[ -e /etc/razor/shiro.ini ]] && mkdir -p /tmp/.razor-server.upgrade && cp /etc/razor/shiro.ini /tmp/.razor-server.upgrade/shiro.ini",
+      "[[ -e #{settings[:configdir]}/config.yaml ]] && mkdir -p /tmp/.razor-server.upgrade && cp #{settings[:configdir]}/config.yaml /tmp/.razor-server.upgrade/config.yaml",
+      "[[ -e #{settings[:sysconfdir]}/shiro.ini ]] && mkdir -p /tmp/.razor-server.upgrade && cp #{settings[:sysconfdir]}/shiro.ini /tmp/.razor-server.upgrade/shiro.ini",
```

Comment:
Are these intentionally moving to different directories?

Reply:
yes. @smcclellan doesn't want the config under /etc for PE since it's managed by the module, so I added a separate configdir setting.

Reply:
ok.